packages/cli-platform-ios/src/tools/pods.ts

```diff
@@ -23,13 +23,13 @@ export function getPackageJson(root: string) {
   return require(path.join(root, 'package.json'));
 }
 
-export function getIosDependencies(
-  dependencies: NativeDependencies,
-  dependenciesVersions: Record<string, string>,
-) {
+export function getIosDependencies(dependencies: NativeDependencies) {
   return Object.keys(dependencies)
     .filter((dependency) => dependencies[dependency].platforms.ios)
-    .map((dependency) => `${dependency}@${dependenciesVersions[dependency]}`)
+    .map(
+      (dependency) =>
+        `${dependency}@${dependencies[dependency].platforms.ios?.version}`,
```

Comment:
`ios?.version` can evaluate to undefined and will produce messy entries

Reply:
since it's already filtered by the dependencies where `platforms.ios` exists, this should never happen. Removed `?` and casted it to `IOSDependencyConfig`